flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/SqlGateway.java

```diff
@@ -168,7 +168,6 @@ public ShutdownThread(SqlGateway gateway) {
         @Override
         public void run() {
             // Shutdown the gateway
-            System.out.println("\nShutting down the Flink SqlGateway...");
```

Comment:
Is this change expected?

Reply:
@libenchao Yes, it will cause e2e test failed because the e2e test framework will check the `.out` log file and confirm that it is empty. I have confirmed this with @fsk119 that the `System.out.println` here is meaningless, we can remove them directly here.

Reply:
sounds good to me.